ProjectSidewalk/SidewalkWebpage
# pano date properly updates

Resolves #3304 

Pano date updates correctly, had to change one line regarding which field to grab to mark as the date.

##### Before/After screenshots

Before:

![initial_validate_wrong](https://github.com/ProjectSidewalk/SidewalkWebpage/assets/56641275/7b85376c-3ce0-44e5-a371-e7a0096f92c0)

![initial_validate_wrong_maps](https://github.com/ProjectSidewalk/SidewalkWebpage/assets/56641275/86ce9d1f-c2db-4efd-92ff-3c96c5e42ae3)

After:

<img width="1090" alt="Screenshot_2023-07-24_at_10 30 14_PM" src="https://github.com/ProjectSidewalk/SidewalkWebpage/assets/56641275/1df86109-61f7-44fb-8676-20aba38041bf">

<img width="1177" alt="Screenshot_2023-07-24_at_10 30 38_PM" src="https://github.com/ProjectSidewalk/SidewalkWebpage/assets/56641275/947424b3-1bbe-42b4-b080-90c1769bb7e4">


##### Testing instructions
1.  Go to Validate page
2. Look at pano date
3. Verify if pano date matches the one on Google Maps

##### Things to check before submitting the PR 

- [????] I've tested on mobile (only needed for validation page)
